### PR TITLE
Add Denon icon

### LINKED ISSUE #5191: Request: Denon Icon
**Brand Name:** Denon
**Website:** https://www.denon.com/en-us/
**Alexa rank:** 51,955
  

**Official resources for icon and color:**
https://en.wikipedia.org/wiki/File:Denon_logo.svg

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
denon
**Issue:** closes #5191 

**Similarweb rank:** 56,972. See https://www.similarweb.com/website/denon.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website header https://www.denon.com
Color: `#FFFFFE` Taken from logo

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3078,6 +3078,12 @@
                 "type": "MIT"
             }
         },
+        {
+            "title": "Denon",
+            "hex": "FFFFFE",
```
Comment (adamrusted) [inline _data/simple-icons.json:3083]:
```suggestion
            "hex": "0B131A",
```
I'd go with this personally, most of their products are black, and they use this in the site header and various social channels.

**Review by adamrusted — COMMENTED** [review]
Icon is spot on, thanks @PratyushJayachandran 🎉
Just the change to the JSON for me - though I'd be happy with either tbh.

**Comment by PratyushJayachandran** [comment]
> Icon is spot on, thanks @PratyushJayachandran 🎉 Just the change to the JSON for me - though I'd be happy with either tbh.

Apples your suggestion. Thank you for reviewing 😊

**Review by adamrusted — APPROVED** [review]
Boom! Let's get this merged! Thanks @PratyushJayachandran 🎉

**Pull request merged**